quantum/quantum_keycodes.h

```diff
@@ -604,6 +604,14 @@ enum quantum_keycodes {
     SECURE_REQUEST,
 
     CAPS_WORD,
+    
```

Comment:
These should go right before safe range.

Reply:
Thanks!
moved!